python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -353,45 +350,49 @@ def warpStamps(self, stamps, pixCenters):
 
         Parameters
         ----------
-        stamps : `collections.abc.Sequence`
-                     [`afwImage.exposure.exposure.ExposureF`]
+        stamps : `Sequence` [`lsst.afw.image.exposure.ExposureF`]
             Image cutouts centered on a single object.
-        pixCenters : `collections.abc.Sequence` [`geom.Point2D`]
-            Positions of each object's center (as obtained from the refCat),
-            in pixels.
+        pixCenters : `Sequence` [`lsst.geom.Point2D`]
+            Positions of each object's center (from the refCat) in pixels.
 
         Returns
         -------
         result : `lsst.pipe.base.Struct`
             Results as a struct with attributes:
 
             ``warpedStars``
-                List of stamps of warped stars (`list` of `afwImage.maskedImage.maskedImage.MaskedImage`).
+                List of stamps of warped stars.
+                    (`list` of `lsst.afw.image.maskedImage.MaskedImage`)
```

Comment:
These long classes could be abbreviate in the rendered docstring by having a tilde in the front.

Reply:
Like this:
```
``warpedStars``
Stamps of warped stars. `list` [`~lsst.pipe.base.PipelineTask`]
```

Reply:
Just saw your suggestion below - modified as you suggest :+1:

Reply:
This is even better